src/plugins/kibana_react/public/context/context.test.tsx

```diff
@@ -27,9 +29,11 @@ afterEach(() => {
 test('can mount <Provider> without crashing', () => {
   const services = coreMock.createStart();
   ReactDOM.render(
-    <context.Provider value={{ services } as any}>
-      <div>Hello world</div>
-    </context.Provider>,
+    <KibanaThemeProvider theme$={theme.theme$}>
```

Comment:
I am not sure I understand the reason why do we need to update tests that won't ever be using `useEuiTheme`, seems like this on is such a test, same for 
`src/plugins/kibana_react/public/context/context.test.tsx`
`src/plugins/kibana_react/public/adapters/ui_to_react_component.test.tsx`
`src/plugins/kibana_react/public/ui_settings/use_ui_setting.test.tsx` 

These are examples of test which don't use components inside that could potentially use `useEuiTheme`, so we won't ever need them to be wrapped in this provider.

I suggest we'd try to have less noise and not wrap where we won't need it

Reply:
also `src/plugins/kibana_utils/common/state_containers/create_state_container_react_helpers.test.tsx`

Reply:
I'd just focus on failing tests